docs/ABOUT.md

```diff
@@ -1,30 +1,16 @@
-Haskell is a purely functional, lazy, statically-typed programming language with type inference.
-
-**Functional** means that functions are first-class data types.
-
-**Purely Functional** means (roughly) that there are no side effects.
-Every function will always return the same value for a given argument will do nothing else.
-
-**Lazy** (a.k.a "non-strict") means that the compiler will put off evaluating a thing until absolutely neccessary.
-This lets you safely do weird stuff like operating on an infinite list--the language will only create it up to the last value you actually use.
-
-**Statically-typed** means that identifiers have a type set at compile time--like those in Java, C++ or C#--instead of holding data of any type like those in Python, Ruby or JavaScript.
-
-**Type inference** means that the compiler will often figure out the type of an identifier by itself so you don't have to specify it.
-Scala and later versions of C# both do this.
-
-Haskell syntax is beautiful and minimal.  For example, here is an
-implemention of quicksort in 6 lines:
-
-```haskell
-quicksort :: (Ord a) => [a] -> [a]
-quicksort [] = []
-quicksort (x:xs) =
-    let smallerSorted = quicksort [a | a <- xs, a <= x]
-        biggerSorted  = quicksort [a | a <- xs, a > x]
-    in  smallerSorted ++ [x] ++ biggerSorted
-```
-[(source)](http://learnyouahaskell.com/recursion#quick-sort)
-
-In addition, Haskell is standardized and has multiple high-quality implementations, some of which produce standalone native binaries.
-There is also a collection of free third-party libraries available and a package manager ("cabal") to automatically fetch them for you.
+Haskell is a general-purpose programming language known for being purely functional, non-strict with strong static typing and for having type inference.
+
+**Purely functional** means that you don't update variables or modify state.
+A function will always return the same value given the same input and will do nothing else.
+**Non-strict** (somewhat like [lazy](https://wiki.haskell.org/Lazy_vs._non-strict)) means that you can express infinite data structures.
+**Strong static typing** means that a lot of program errors are caught during compilation.
+**Type inference** means that the compiler can often figure out the type of a value by itself.
+The compiler can also tell you if a value has conflicting types in different parts of the code.
+
+There are more than 10,000 free third-party packages available at
```

Comment:
I think this needs to be reflowed to be one line per sentence. The purpose of one line per sentence stated in https://github.com/exercism/haskell/issues/686#issuecomment-413021257 is to make diffs better, but I think the line breaks here don't reflect the divisions at which someone might want to make changes

Reply:
Done.